rules/falco_rules.yaml

```diff
@@ -2315,6 +2315,9 @@
 - macro: user_known_user_management_activities
   condition: (never_true)
 
+- macro: chage_list
+  condition: (proc.name=chage and (proc.cmdline contains "-l" or proc.cmdline contains "--list"))
```

Comment:
Are we assuming that `chage` blocks the usage of other flags when `-l` or `--list` is specified?

eg.
```
$ chage -l -m 10 leogr
chage: do not include "l" with other flags
Usage: chage [options] LOGIN
....
```

Reply:
Mainly for whitelisting the `chage -l` or `chage --list` can be conducted for regular check purpose.